app/src/main/res/drawable/circle_accent.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+  android:shape="oval">
+  <solid android:color="@color/accent" />
+  <padding
+    android:bottom="@dimen/grid_1_half"
```

Comment:
could you use `android:padding="@dimen/grid_1_half" `

Reply:
yeah! will update

Reply:
Actually, you cannot! You have to specify each side https://developer.android.com/guide/topics/resources/drawable-resource#Shape